opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/services/ProtocolResultHandler.java

```diff
@@ -235,6 +242,11 @@
         return mappedResult;
     }
 
+    private void setAspspRedirectTokenIfRequired(String authKey, FacadeRedirectResult mappedResult) {
+        if (StringUtils.isNotBlank(authKey)) {
```

Comment:
Use Guava's `Strings.isNullOrEmpty()` it makes no sense to use commons as a dependency just for 1 function

Reply:
done